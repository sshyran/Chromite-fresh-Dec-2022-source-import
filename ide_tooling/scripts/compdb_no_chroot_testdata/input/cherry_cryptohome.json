[
  {
    "command": "armv7a-cros-linux-gnueabihf-clang++ -MMD -MF obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include/bindings -Igen/include -I../../../../../../../tmp/portage/chromeos-base/cryptohome-9999/work/cryptohome-9999/platform2 -I/build/cherry/usr/include -Igen -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/build/cherry -fPIE -fPIC -Wno-unreachable-code -Wno-deprecated-declarations -std=gnu++17 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c gen/include/bindings/install_attributes.pb.cc -o obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o",
    "directory": "/build/cherry/var/cache/portage/chromeos-base/cryptohome/out/Default",
    "file": "gen/include/bindings/install_attributes.pb.cc",
    "output": "obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o"
  },
  {
    "command": "armv7a-cros-linux-gnueabihf-clang++ -MMD -MF obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include/bindings -Igen/include -I../../../../../../../tmp/portage/chromeos-base/cryptohome-9999/work/cryptohome-9999/platform2 -I/build/cherry/usr/include -Igen -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/build/cherry -fPIE -fPIC -Wno-unreachable-code -Wno-deprecated-declarations -std=gnu++17 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c gen/include/bindings/chrome_device_policy.pb.cc -o obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o",
    "directory": "/build/cherry/var/cache/portage/chromeos-base/cryptohome/out/Default",
    "file": "gen/include/bindings/chrome_device_policy.pb.cc",
    "output": "obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o"
  }
]
