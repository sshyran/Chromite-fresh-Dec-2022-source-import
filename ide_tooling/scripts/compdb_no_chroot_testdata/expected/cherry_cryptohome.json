[
  {
    "command": "clang++ --target=arm -MMD -MF obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include/bindings -Igen/include -I/usr/local/google/home/oka/os2/src/platform2 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include -Igen -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/usr/local/google/home/oka/os2/chroot/build/cherry -fPIE -fPIC -Wno-unreachable-code -Wno-deprecated-declarations -std=gnu++17 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c gen/include/bindings/install_attributes.pb.cc -o obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o -stdlib=libc++",
    "directory": "/usr/local/google/home/oka/os2/chroot/build/cherry/var/cache/portage/chromeos-base/cryptohome/out/Default",
    "file": "gen/include/bindings/install_attributes.pb.cc",
    "output": "obj/gen/include/bindings/libinstall_attributes-proto.install_attributes.pb.o"
  },
  {
    "command": "clang++ --target=arm -MMD -MF obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o.d -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Igen/include/bindings -Igen/include -I/usr/local/google/home/oka/os2/src/platform2 -I/usr/local/google/home/oka/os2/chroot/build/cherry/usr/include -Igen -Wall -Wunused -Wbool-operation -Wfree-nonheap-object -Wint-in-bool-context -Wstring-compare -Wstring-plus-int -Wxor-used-as-pow -Wimplicit-int-float-conversion -Wno-c99-designator -Wno-unused-parameter -Wunreachable-code -Wunreachable-code-return -ggdb3 -fstack-protector-strong -Wformat=2 -fvisibility=internal -Wa,--noexecstack -Wimplicit-fallthrough -Werror --sysroot=/usr/local/google/home/oka/os2/chroot/build/cherry -fPIE -fPIC -Wno-unreachable-code -Wno-deprecated-declarations -std=gnu++17 -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -Os -pipe -march=armv8-a+crc -mfpu=crypto-neon-fp-armv8 -mfloat-abi=hard -g -ffunction-sections -fdata-sections -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -c gen/include/bindings/chrome_device_policy.pb.cc -o obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o -stdlib=libc++",
    "directory": "/usr/local/google/home/oka/os2/chroot/build/cherry/var/cache/portage/chromeos-base/cryptohome/out/Default",
    "file": "gen/include/bindings/chrome_device_policy.pb.cc",
    "output": "obj/gen/include/bindings/libpolicy-protos.chrome_device_policy.pb.o"
  }
]
